Yandex cloud log formatting changes in 0.142 and above crash info level logs

The change https://github.com/yandex-cloud/python-sdk/pull/33 broke logging functionality.

Yandexloud 0.142, 0.143, 0.144 and 0.145 are broken.

For example it generates this error:

```
/usr/local/lib/python3.7/site-packages/yandexcloud/_wrappers/dataproc/__init__.py:466: in create_hive_job
    self.log.info("Running Hive job. Cluster ID: {cluster_id}", cluster_id=cluster_id)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <RootLogger root (INFO)>, msg = 'Running Hive job. Cluster ID: {cluster_id}', args = (), kwargs = {'cluster_id': 'my_cluster_id'}

    def info(self, msg, *args, **kwargs):
        """
        Log 'msg % args' with severity 'INFO'.
    
        To pass exception information, use the keyword argument exc_info with
        a true value, e.g.
    
        logger.info("Houston, we have a %s", "interesting problem", exc_info=1)
        """
        if self.isEnabledFor(INFO):
>           self._log(INFO, msg, args, **kwargs)
E           TypeError: _log() got an unexpected keyword argument 'cluster_id'

/usr/local/lib/python3.7/logging/__init__.py:1378: TypeError
---------------------------------------------------------------------------------
```

The problem is that Python logging does not support "brace" style logging by default and unless you configure your application, it only supports "%" style logging. 

See https://docs.python.org/3/howto/logging-cookbook.html#using-particular-formatting-styles-throughout-your-application

Brace style parameters should not be used in libraries, because you never know how customers of your library will use it.

--- a/.pylintrc
+++ b/.pylintrc
@@ -14,3 +14,4 @@ disable=
     R0903, # Too few public methods (1/2) (too-few-public-methods)
     R0913, # Too many arguments (6/5) (too-many-arguments)
     R1710, # Either all return statements in a function should return an expression, or none of them should. (inconsistent-return-statements)
+    W1202, # Use lazy % formatting in logging functions (logging-format-interpolation)

--- a/yandexcloud/_wrappers/dataproc/__init__.py
+++ b/yandexcloud/_wrappers/dataproc/__init__.py
@@ -353,7 +353,7 @@ class Dataproc(object):
             disk_type_id=disk_type,
         )
 
-        self.log.info("Adding subcluster to cluster {cluster_id}", cluster_id=cluster_id)
+        self.log.info("Adding subcluster to cluster {cluster_id}".format(cluster_id=cluster_id))
         autoscaling_config = None
         if max_hosts_count:
             autoscaling_config = subcluster_pb.AutoscalingConfig(
@@ -395,7 +395,7 @@ class Dataproc(object):
         if not cluster_id:
             raise RuntimeError("Cluster id must be specified.")
 
-        self.log.info("Updating cluster {cluster_id}", cluster_id=cluster_id)
+        self.log.info("Updating cluster {cluster_id}".format(cluster_id=cluster_id))
         mask = FieldMask(paths=["description"])
         request = cluster_service_pb.UpdateClusterRequest(
             cluster_id=cluster_id,
@@ -420,7 +420,7 @@ class Dataproc(object):
         if not cluster_id:
             raise RuntimeError("Cluster id must be specified.")
 
-        self.log.info("Deleting cluster {cluster_id}", cluster_id=cluster_id)
+        self.log.info("Deleting cluster {cluster_id}".format(cluster_id=cluster_id))
         request = cluster_service_pb.DeleteClusterRequest(cluster_id=cluster_id)
         return self.sdk.create_operation_and_get_result(
             request,
@@ -463,7 +463,7 @@ class Dataproc(object):
             raise RuntimeError("Cluster id must be specified.")
         if (query and query_file_uri) or not (query or query_file_uri):
             raise RuntimeError("Either query or query_file_uri must be specified.")
-        self.log.info("Running Hive job. Cluster ID: {cluster_id}", cluster_id=cluster_id)
+        self.log.info("Running Hive job. Cluster ID: {cluster_id}".format(cluster_id=cluster_id))
 
         hive_job = job_pb.HiveJob(
             query_file_uri=query_file_uri,
@@ -530,7 +530,7 @@ class Dataproc(object):
         cluster_id = cluster_id or self.cluster_id
         if not cluster_id:
             raise RuntimeError("Cluster id must be specified.")
-        self.log.info("Running Mapreduce job. Cluster ID: {cluster_id}", cluster_id=cluster_id)
+        self.log.info("Running Mapreduce job. Cluster ID: {cluster_id}".format(cluster_id=cluster_id))
 
         request = job_service_pb.CreateJobRequest(
             cluster_id=cluster_id,
@@ -602,7 +602,7 @@ class Dataproc(object):
         cluster_id = cluster_id or self.cluster_id
         if not cluster_id:
             raise RuntimeError("Cluster id must be specified.")
-        self.log.info("Running Spark job. Cluster ID: {cluster_id}", cluster_id=cluster_id)
+        self.log.info("Running Spark job. Cluster ID: {cluster_id}".format(cluster_id=cluster_id))
 
         request = job_service_pb.CreateJobRequest(
             cluster_id=cluster_id,
@@ -677,7 +677,7 @@ class Dataproc(object):
         cluster_id = cluster_id or self.cluster_id
         if not cluster_id:
             raise RuntimeError("Cluster id must be specified.")
-        self.log.info("Running Pyspark job. Cluster ID: {cluster_id}", cluster_id=cluster_id)
+        self.log.info("Running Pyspark job. Cluster ID: {cluster_id}".format(cluster_id=cluster_id))
         request = job_service_pb.CreateJobRequest(
             cluster_id=cluster_id,
             name=name,

